tests/kuttl/tests/basic/01-deploy-heat.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+    PWD=$INSTALL_YAMLS
```

Comment:
seems yamllint is complaining the wrong format at this like. Probably you should add one more indent ?